angular-chess/src/app/components/chess-board.component.ts

```diff
@@ -18,7 +18,12 @@ import PositionUtils from '../utils/position.utils';
   styleUrls: ['./chess-board.component.scss'],
   providers: [MessageService]
 })
-export class ChessBoardComponent {
+export class ChessBoardComponent implements OnInit {
+  public readonly numbersOneToEight: number[] = [1, 2, 3, 4, 5, 6, 7, 8];
```

Comment:
It can be done with a function:
```suggestion
  public readonly numbersOneToEight: number[] = [...Array(8)].map((_, i) => i + 1);
```